components/search/FilterSidebar.js

```diff
@@ -49,8 +49,9 @@ class FilterSidebar extends React.Component {
   constructor(props) {
     super(props)
 
-    // Display `${today}` as the end date for default search
-    const today = moment().format('YYYY-MM-DD')
+    // Display `${tomorrow}` as the end date for default search
+    // to include the measurements of `${today}` as well.
+    const until = moment().add(1, 'day').format('YYYY-MM-DD')
```

Comment:
Is this `until` date going to be the current date in UTC + 1?

It's worth considering that the `until` filter in the API uses the time in UTC, while I suspect that `moment()` will by default use the timezone of the browser.

We should take that into account and here compute 1 day from now in UTC.

Reply:
This is a very good point! So, we need to use `moment().utc()` according to https://momentjs.com/docs/#/parsing/utc/.

Reply:
This is a good point. I will send an updated PR for the same. Also, this makes me think, shouldn't all the date (like `since`) be also in UTC. Otherwise that also would give based on timezone.

Reply:
Yes, I believe there is a bunch of places where we're not using UTC and we should probably make sure we're using the proper time in the proper context.

I filed https://github.com/ooni/explorer/issues/368 to track this localtime->UTC work.